### PR TITLE
Add monochrome icons for appindicator

### PR DESCRIPTION
**Pull request by purejava**
Add talk-name for appindicator

### CODE REVIEW

**Comment by purejava** [comment]
This packs `libappindicator` within the flatpacked app and therefore enables tray icons for Cryptomator.

The [flathub/shared-modules](https://github.com/flathub/shared-modules) are required. Once this repo is cloned, `git submodule add https://github.com/flathub/shared-modules.git` does include these.
I don't know whether the shared modules are available on flathub or if further configuration is required to include them there.

The build currently fails, as the Cryptomator package including the icons is not published in the Cryptomator repo yet and the maven dependency files are not updated in this repo yet.

File: org.cryptomator.Cryptomator.yaml [hunk 1]
```diff
@@ -124,9 +131,14 @@ modules:
...
       - cp -R Cryptomator /app/
       - ln -s /app/Cryptomator/bin/Cryptomator /app/bin/cryptomator
+      - cp -R /app/lib/* /app/Cryptomator/lib/app/
       - install -D -m0644 -t /app/share/applications/ dist/linux/common/org.cryptomator.Cryptomator.desktop
       - install -D -m0644 -t /app/share/icons/hicolor/scalable/apps/ dist/linux/common/org.cryptomator.Cryptomator.svg
+      - install -D -m0644 -t /app/share/icons/hicolor/symbolic/apps/ dist/linux/common/org.cryptomator.Cryptomator.tray.svg
+      - install -D -m0644 -t /app/share/icons/hicolor/symbolic/apps/ dist/linux/common/org.cryptomator.Cryptomator.tray-unlocked.svg
       - install -D -m0644 -t /app/share/metainfo/ dist/linux/common/org.cryptomator.Cryptomator.metainfo.xml
+      - mv /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray.svg /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray-symbolic.svg
+      - mv /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray-unlocked.svg /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray-unlocked-symbolic.svg
```
Comment (overheadhunter) [inline org.cryptomator.Cryptomator.yaml:141]: Why not installing them to this location in the first place?

Comment (purejava) [reply org.cryptomator.Cryptomator.yaml:141]: It's renaming them and adding `-symbolic` to the filename.

The files exist in `common` without `-symbolic` in their filenames and are copied from there. This avoids having the same files doubled in `common` just with different filenames.

Comment (overheadhunter) [reply org.cryptomator.Cryptomator.yaml:141]: yes but can't we copy and rename those in a single step?

```suggestion
      - install -D -m0644 -T dist/linux/common/org.cryptomator.Cryptomator.tray.svg /app/share/icons/hicolor/symbolic/apps/org.cryptomator.Cryptomator.tray-symbolic.svg
```

Comment (purejava) [reply org.cryptomator.Cryptomator.yaml:141]: Yep. I tried it with `-t` in the first place, but that created directories. It needs to be `-T`, as you mentioned.

**Comment by purejava** [comment]
Hi @infeo,

build currently fails as the [shared modules](https://github.com/flathub/org.cryptomator.Cryptomator/pull/39#issuecomment-1574963330) are missing. I did not provide a solution on embedding these to the CI build. 

After adding the modules and building the flatpaked Cryptomator, I noticed that app fails to load the native libs:
```java
19:10:15.412 [JavaFX Application Thread] INFO  o.p.appindicator.RuntimeHelper - Native code library libayatana-appindicator3.so.1 failed to load
19:10:15.412 [JavaFX Application Thread] INFO  o.p.appindicator.RuntimeHelper - Native code library libappindicator3.so.1 failed to load
```

I'll open a separate issue for this.

**Comment by purejava** [comment]
> Hi @infeo,
> 
> build currently fails as the [shared modules](https://github.com/flathub/org.cryptomator.Cryptomator/pull/39#issuecomment-1574963330) are missing. I did not provide a solution on embedding these to the CI build.
> 

Fixed.

**Pull request merged**